lib/ex_unit_fixtures.ex

```diff
@@ -99,6 +99,37 @@ defmodule ExUnitFixtures do
   alias ExUnitFixtures.FixtureDef
 
   @doc """
+  Starts the ExUnitFixtures application.
+  """
+  def start() do
+    Application.ensure_all_started(:ex_unit_fixtures)
+  end
+
+  @doc false
+  def start(_type, _args) do
+    import Supervisor.Spec, warn: false
+
+    children = [
+      worker(ExUnitFixtures.Imp.ModuleStore, []),
```

Comment:
It might be good to call `load_fixture_files` automatically, rather than requiring the user to explicitly call it.  Either by defining a worker that does that in it's init, or just by calling it after the start_link call.